boms/cloud-oss-bom/RELEASING.md

```diff
@@ -14,18 +14,20 @@ Create a PR for the release, and get it approved.
 ## Rapid build
 
 Rapid project is [cloud-java-tools-cloud-opensource-java-bom-kokoro-release](
-http://rapid/cloud-java-tools-cloud-opensource-java-bom-kokoro-release).
+https://rapid/cloud-java-tools-cloud-opensource-java-bom-kokoro-release).
```

Comment:
This isn't right. `https` URLs shouldn't use simple domain names. I'd switch back to `http`.

Reply:
done